web/thesauruses/python/3/lists.json

```diff
@@ -0,0 +1,492 @@
+      ],
+      "not-implemented": "true"
+    },
+    "delete_mutable_hashed_list": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "name_of_immutable_hashed_list": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "create_a_immutable_hashed_list": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "insert_element_to_immutable_hashed_list": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "erase_element_from_immutable_hashed_list": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "delete_immutable_hashed_list": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "create_a_mutable_set": {
+      "code": [
+        "dict_name = {}",
+        "dict_name = {key: value}",
+        "dict_name = {key1: value1, key2: value2, ...}"
+      ],
+      "comment": "First line creates an empty dict. Second line creates a dict with one key/value pair. Third line creates a dict with multiple key/value pairs"
+    },
+    "get_key_from_mutable_set": {
+      "code": [
+        "list(dict_name.keys())[index]"
+      ]
+    },
+    "get_value_from_mutable_set": {
+      "code": [
+        "dict_name[key]",
+        "dict_name.get(key)",
+        "dict_name.get(key, default_value)"
+      ],
+      "comment": "If the key cannot be found, .get(key) returns a default value (or None if not specified), whereas [key] will raise an exception"
+    },
+    "get_all_keys_from_mutable_set": {
+      "code": [
+        "dict_name.keys()"
+      ]
+    },
+    "get_all_values_from_mutable_set": {
+      "code": [
+        "dict_name.values()"
+      ]
+    },
+    "swap_key_and_value_in_mutable_set": {
+      "code": [
+        "dict_name = dict([(value, key) for key, value in dict_name.items()])"
+      ]
+    },
+    "delete_mutable_set": {
+      "code": [
+        "del dict_name"
+      ]
+    },
+    "create_a_immutable_set": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "get_key_from_immutable_set": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "get_value_from_immutable_set": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "get_all_keys_from_immutable_set": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "get_all_values_from_immutable_set": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "swap_key_and_value_in_immutable_set": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "delete_immutable_set": {
+      "code": [
+        ""
+      ],
+      "not-implemented": "true"
+    },
+    "find_element_by_value": {
+      "code": [
+        "value in list_name"
+      ],
+      "comment": "returns a True/False value"
+    },
```

Comment:
I believe this should be `list_name.index(value)`

Reply:
Could have both since they both find by value. Would comment that one returns a boolean and the other returns an index/integer